docs/for-ops/how-to/change-admin-password.md

```diff
@@ -0,0 +1,119 @@
+---
+slug: change-admin-password
+title: Change the otomi-admin password
+sidebar_label: Change admin password
+---
+
+This how to provides step-by-step instructions to reset the `otomi-admin` password.
+
+## Prerequisites:
+
+1. Access to the Keycloak admin console.
+
+2. Docker  installed and configured.
+
+3. SOPS set up with Age encryption (default in LKE).
+
+## Steps:
+
+### Step 1: Generate a New Password
+
+```bash
+    # Generate a random 16-character alphanumeric password
+    head /dev/urandom | tr -dc A-Za-z0-9 | head -c 16
```

Comment:
This command doesn't work for me
`tr: Illegal byte sequence`
You can also say generate a uuid? Which can be done with:
`uuidgen`

Reply:
Updated existing command. Not it should work with MacOS coreutils.